pkgs/by-name/ai/airscan/package.nix

```diff
@@ -0,0 +1,25 @@
+{ buildGoModule
+, fetchFromGitHub
+, lib
+}:
+
+buildGoModule rec {
+  pname = "airscan";
+  version = "0.3";
+
+  src = fetchFromGitHub {
+    owner = "stapelberg";
+    repo = pname;
```

Comment:
```suggestion
    repo = "airscan";
```

Reply:
The change suggests the way which is preferred nowadays.

Reply:
These bikeshedding comments are extremely annoying, especially because a certain other committer was running around for two years commenting the opposite on each PR.

Reply:
If you want to change everyone's opinion on this, at least link a discussion with some more information about how this new consensus was reached.

Reply:
Otherwise it's the following experience:
- I do x
- people tell to do y instead
- I do y
- people tell to do x instead